deploy/kpng-windows-capz-calico.yaml

```diff
@@ -51,7 +52,7 @@ spec:
             # CP node directly since we don't have routes programmed yet.
             # NOTE: These values are specific to the default CAPZ deployment.
             - name: KUBERNETES_SERVICE_HOST
-              value: "10.0.0.4"
+              value: "10.20.30.10"
```

Comment:
did the default value for CAPZ clusters change here?

Reply:
reverting /